pulp_rpm/app/serializers/prune.py

```diff
@@ -0,0 +1,81 @@
+from gettext import gettext as _
+
+from rest_framework import fields, serializers
+
+from pulp_rpm.app.models import RpmRepository
+
+from pulpcore.plugin.serializers import ValidateFieldsMixin
+from pulpcore.plugin.util import get_domain
+
+
+class PruneNEVRAsSerializer(serializers.Serializer, ValidateFieldsMixin):
```

Comment:
`PrunePackagesSerializer`

Reply:
I don't like the idea of exposing NEVRA terminology directly to the user too much, it's ok in log messages and such but it's not the most clear terminology to use in a circumstance like this.

Reply:
Changing - but I'll point out that these are largely code-objects that are only ever seen by pulp_rpm developers. Also - if you're calling this functionality, **you specifically are basing your prune-strategy on EVR**. But I'm easily swayed :)